src/EnergyPlus/ChillerElectricEIR.cc

```diff
@@ -2296,6 +2350,70 @@ void ElectricEIRChillerSpecs::calculate(EnergyPlusData &state, Real64 &MyLoad, b
+                    std::abs(state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).LoopSide(this->CWPlantLoc.loopSideNum).UpdatedDemandToLoopSetPoint);
+                Real64 cwhLoopPLR = 0.0;
+                if (chwLoopDemand > 0) {
+                    cwhLoopPLR = chwLoopDemand / chwLoopCap;
+                }
+                Real64 condWaterFlowFrac = Curve::CurveValue(state, this->ChillerCondLoopFlowFLoopPLRIndex, cwhLoopPLR);
+                Real64 cwLoopDesVolFlowRate = state.dataSize->PlantSizData(CondPltSizNum).DesVolFlowRate;
+                Real64 cwLoopVolFlowRate = condWaterFlowFrac * cwLoopDesVolFlowRate;
+                Real64 rho = FluidProperties::GetDensityGlycol(state,
+                                                               state.dataPlnt->PlantLoop(this->CDPlantLoc.loopNum).FluidName,
+                                                               this->TempRefCondIn,
+                                                               state.dataPlnt->PlantLoop(this->CDPlantLoc.loopNum).FluidIndex,
+                                                               RoutineName);
+                if (chwLoopDemand > 0) {
+                    this->CondMassFlowRate = cwLoopVolFlowRate * rho * this->QEvaporator / chwLoopDemand;
+                } else {
+                    this->CondMassFlowRate = 0.0;
+                }
+            } else {
+                ShowFatalError(state,
+                               format("{}: The ModulatedLoopPLR condenser flow control requires a Sizing:Plant object for "
+                                      "both loops connected to the condenser and evaporator of the chiller.",
+                                      RoutineName));
+            }
+        } break;
+        case DataPlant::CondenserFlowControl::ModulatedDeltaTemperature: {
+            Real64 Cp = FluidProperties::GetSpecificHeatGlycol(state,
+                                                               state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).FluidName,
+                                                               this->CondInletTemp,
+                                                               state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).FluidIndex,
+                                                               RoutineName);
+            Real64 condDT = 0.0;
+            if (this->CondDTScheduleNum > 0) {
+                condDT = ScheduleManager::GetCurrentScheduleValue(state, this->CondDTScheduleNum);
+            }
+            this->CondMassFlowRate = this->QCondenser / (Cp * condDT);
+        } break;
+        default: {
+            this->CondMassFlowRate = this->CondMassFlowRateMax;
+        } break;
+        }
+        this->CondMassFlowRate = max(min(this->CondMassFlowRate, this->CondMassFlowRateMax),
+                                     this->MinCondFlowRatio * this->CondMassFlowRateMax,
+                                     this->VSBranchPumpMinLimitMassFlowCond);
```

Comment:
Apply the user-specified minimum flow fraction and consider the pump minimum flow rate as well.

Reply:
So this is the max of three things?  I wasn't expecting that.  The logic here is fine, but I wouldn't mind this broken into an intermediate variable or two.  The combined min/max with multiplication in the middle makes it difficult to spot a bug.  The MinCondFlowRatio * CondMassFlowRateMax could be a single variable.  Then the min(.., ..) portion could be a single named variable.  Then the max of all three would be clear.  This would be a nice change.